lib/beautiful/xresources.lua

```diff
@@ -83,8 +83,16 @@ function xresources.get_dpi(s)
         if awesome and awesome.xrdb_get_value then
             xresources.dpi = tonumber(awesome.xrdb_get_value("", "Xft.dpi"))
```

Comment:
I meant the comment added to the function in the actual commit. I can rephrase the commit message to include the link itself to make it clearer.